agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -213,6 +213,22 @@ class EvaluationScenarioOutput(EmbeddedModel):
     variant_output: str
 
 
+class BulkEvaluationDB(Model):
+    app: AppDB = Reference(key_name="app")
+    organization: OrganizationDB = Reference(key_name="organization")
+    user: UserDB = Reference(key_name="user")
+    status: str
+    evaluation_type: List[str]
+    evaluation_type_settings: EvaluationTypeSettings
+    variants: List[ObjectId]
+    testset: TestSetDB = Reference(key_name="testsets")
```

Comment:
Since you're setting the key_name of the **testset** field to **testsets**, does this mean that the field is going to be storing multiple **testsets**? If yes, then I think it'll be advisable to refactor the field to this:

```suggestion
    testsets: List[ObjectId]
```

Otherwise, it is fine to call the Reference object with no default value.

Reply:
I think one eval should link to one test set

Reply:
we might allow running an eval in one batch on multiple testsets, but no reason to save all of these in one documetn